app/views/home/index.html.haml

```diff
@@ -1,2 +1,84 @@
+.max-w-5xl.mx-auto.lg:grid.grid-cols-1.lg:grid-cols-12.gap-x-12.gap-y-4.relative.z-20
+  .lg:col-span-12.text-center
+    .flex.justify-center.mb-4
+      = image_tag "plant_logo.png", size: "200"
+    %h1.mb-4.text-5xl.font-semibold.text-gray-800.text-center.mt-3 PlantzApp
+    %p.leading-snug.text-gray-700.text-lg.max-w-3xl.mx-auto.pb-6 Find new house plants, ask the community and get personalized watering reminders.
+
+  .lg:col-span-12.mb-5
+    %h2.font-semibold.text-2xl.mb-4 Houseplants
+    %ul.flex.flex-wrap.items-center.text-sm.font-medium.text-center.text-white.border-b-2.border-emerald-600
+      %li.mr-2
+        %a.inline-block.p-4.bg-emerald-700.active:bg-emerald-700{"aria-current" => "page", href: "#"} All
+      %li.mr-2
+        %a.inline-block.p-4.text-black.bg-emerald-300.hover:bg-emerald-400{href: "#"} Top 10
+      %li.mr-2
+        %a.inline-block.p-4.text-black.bg-emerald-300.hover:bg-emerald-400{href: "#"} Categories
+      %li.ml-auto
+        %button.text-black.p-3.flex.items-center.hover:bg-emerald-400
+          -# = link_to "plants" plants
+          %svg.mr-2{fill: "#047857", height: "28", version: "1.1", viewbox: "0 0 26 28", width: "26", xmlns: "http://www.w3.org/2000/svg"}
+            %title search1
+            %path{d: "M18 13c0-3.859-3.141-7-7-7s-7 3.141-7 7 3.141 7 7 7 7-3.141 7-7zM26 26c0 1.094-0.906 2-2 2-0.531 0-1.047-0.219-1.406-0.594l-5.359-5.344c-1.828 1.266-4.016 1.937-6.234 1.937-6.078 0-11-4.922-11-11s4.922-11 11-11 11 4.922 11 11c0 2.219-0.672 4.406-1.937 6.234l5.359 5.359c0.359 0.359 0.578 0.875 0.578 1.406z"}
+          %p.text-base Search for plants
+
+    .flex.justify-center.w-full.h-min.sm:pt-8
+      %button#prev.-ml-10.focus:outline-none.focus:bg-gray-400.focus:ring-2.focus:ring-offset-2.focus:ring-gray-400.cursor-pointer{"aria-label" => "slide backward"}
+        %svg{fill: "none", height: "20", viewbox: "0 0 8 14", width: "12", xmlns: "http://www.w3.org/2000/svg"}
+          %path{d: "M7 1L1 7L7 13", stroke: "currentColor", "stroke-linecap" => "round", "stroke-linejoin" => "round", "stroke-width" => "2"}
+      .w-full.h-min.relative.mx-auto.overflow-x-hidden.overflow-y-hidden
+        #slider.h-80.w-80.flex.lg:gap-8.md:gap-6.gap-14.justify-start.transition.ease-out.duration-700
+          .flex.flex-shrink-0.relative.w-full.sm:w-auto
+            %img.object-cover.object-center.w-full{alt: "black chair and white table", src: "http://127.0.0.1:3000/rails/active_storage/representations/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBEQT09IiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--2489c0392ed9f07c96acb7a591fcf6bf634c67a4/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDVG9MWm05eWJXRjBTU0lJYW5CbkJqb0dSVlE2RTNKbGMybDZaVjkwYjE5bWFXeHNXd2RwQXBBQmFRS1FBVG9NWTI5dWRtVnlkRWtpQ1hkbFluQUdPd1pVT2dwellYWmxjbnNHT2d4eGRXRnNhWFI1YVdrPSIsImV4cCI6bnVsbCwicHVyIjoidmFyaWF0aW9uIn19--449f6a4c65772c435deba3f786d8174e92ec2bfa/2024-Bugatti-W16-Mistral-006-1080.jpg"}/
```

Comment:
As @czepesch rightly said, this will not work in another environment. And will definitely not work in production

Reply:
@pipen95 There should be another for you to reach the images you want here, the links are dynamic so think of a way to dynamically reach it, on whatever environment the application is running.

<img width="1169" alt="Screen Shot 2022-08-28 at 9 06 35 AM" src="https://user-images.githubusercontent.com/32690770/187064348-e2f3d689-9477-4a79-a8b2-306c098ca234.png">